backend/prompts/prompting.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import os
+
+from jinja2 import Environment, FileSystemLoader
+
+
+class PromptEngine:
+    """
+    Class to handle loading and populating Jinja2 templates for prompts.
+    """
+
+    def __init__(self, model: str):
+        """
+        Initialize the PromptEngine with the specified model.
+
+        Args:
+            model (str): The model to use for loading prompts.
+            debug_enabled (bool): Enable or disable debug logging.
```

Comment:
looks like `debug_enabled` isn't used anywhere?

Reply:
True, I've removed it from the init method signature. Removed